src/main/webapp/search.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Piece of Cake!</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="header" class="wrapper">
+      <nav class="navbar">
+          <a href="/index.html" class="link nav-item mx-3">LOGO</a>
+          <form action="/search.html" class="form-line">
+               <input type="text" class="form-control" id="searchbar" placeholder="Type recipe title" name="searchterm">
+               <button type="submit" class="btn btn-secondary mx-3" value="search">search</button>
+          </form>
+          <div id="links">
+            <a class="link nav-item mx-3">post a recipe</a>
+            <a class="link nav-item mx-3">log in/sign up</a>
+          </div>
+      </nav>
+    </div> 
+    <div class="content">
+        <ul id="results" class="list-group list-group-flush">
+            <li class="list-group-item">
+                 <img src="images/default.png" class="img-thumbnail my-3 mr-3 float-left" style="width: 200px; height: 200px" alt="Recipe Image">
```

Comment:
Instead of `style="width: 200px; height: 200px"`, you can set the `width` and `height` properties.

`<img ... width="200" height="200" />`